playbooks/app_install/roles/docker_install/vars/main.yml

```diff
@@ -1,15 +1,35 @@
 ---
 # vars file for docker_install
-# Which docker version should be installed
-docker_version: '23.0.6'
+docker_version: 24.0.6
+# Whether to retrieve from artifactory to push to target
+# or push from the controller to the target when set local
+docker_rpm_src: artifactory
+artifact_path:  "https://engci-maven-master.cisco.com/artifactory/cmsp-sapphire-development-group/up-sac-utils/jenkins/v1/docker-ce-24.0.6-rpms.tar.gz"
```

Comment:
Any reason for using cmsp-sapphire?  Will we have UP's own later down the road?  Also, better to use the docker_version variable name here.